packages/reactotron-server/tsconfig.test.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "sourceMap": false,
     "inlineSourceMap": true,
-    "outDir": "build",
+    "outDir": "build/server",
+    "noUnusedLocals": false,
```

Comment:
Calling this out just to make sure you guys are happy. Note this only applies in tests, not in actual source.